payload-functions.js

```diff
@@ -287,6 +287,29 @@ function prepareCreateSnapshotPayload(params) {
   };
 }
 
+function prepareCreateTags(params) {
```

Comment:
I think we should keep the function names consistent, so this function should be named `prepareCreateTagsPayload`, also it explicitly describes the function's purpose.

Reply:
Yep, my bad. Fixed